inference-engine/tests/functional/inference_engine/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ set(LINK_LIBRARIES
     inference_engine_snippets
     offline_transformations
     inference_engine
-    frontend_manager
+    frontend_common
```

Comment:
create alias and link as is?

Reply:
Wont it confuse users, since there is no such target?

Reply:
Wont it confuse users, since there is no such target?